tests/devapps/WAM/NetFrameworkWam/Form1.cs

```diff
@@ -513,6 +540,11 @@ private async void btnRemoveAcc_Click(object sender, EventArgs e)
                 Log("Exception: " + ex);
             }
         }
+
+        private void useBrokerChk_CheckedChanged(object sender, EventArgs e)
+        {
+
```

Comment:
empty handler?

Reply:
Need to add uncheck "Runtime" when WAM is unchecked,